doc/sdk-cli-reference.md

```diff
@@ -428,7 +428,9 @@ Run scorecard tests on an operator
 ### Flags
 
 * `basic-tests` - Enable basic operator checks (default true)
+* `config` string - config file (default is `<project_dir>/.osdk-yaml`)
```

Comment:
Should the default filename be `<project_dir>/.osdk-scorecard`?

https://github.com/operator-framework/operator-sdk/blob/d4d37b10fc4bd875f56d75d59c69b3e53674a304/internal/pkg/scorecard/scorecard.go#L455

Reply:
Also, by the way, it looks like this should be fixed here as well:

https://github.com/operator-framework/operator-sdk/blob/d4d37b10fc4bd875f56d75d59c69b3e53674a304/cmd/operator-sdk/scorecard/cmd.go#L39

Reply:
Yeah. I think I probably forgot to change the help text when we decided on a default name. I'll fix that.

Reply:
Updated config file usage text